lib/active_merchant/billing/gateways/securion_pay.rb

```diff
@@ -133,13 +133,48 @@ def create_post_for_auth_or_purchase(money, payment, options)
         add_creditcard(post, payment, options)
         add_customer(post, payment, options)
         add_customer_data(post, options)
+        add_three_ds(post, options)
```

Comment:
Let's rename this for specificity.

```suggestion
        add_external_three_ds(post, options)
```